src/Search.php

```diff
@@ -3755,7 +3755,7 @@ public static function addOrderBy($itemtype, $sort_fields, $_id = 'ASC')
                                 $addaltemail = ",
                                 IFNULL(`$ticket_user_table`.`alternative_email`, '')";
                             }
-                            if ($searchopt[$ID]['linkfield'] != 'users_id_lastupdater' && $searchopt[$ID]['linkfield'] != 'users_id_recipient') {
+                            if (!empty($addtable) && !empty($complexjoin)) {
```

Comment:
Maybe the same condition as in `addSelect()` should be used
https://github.com/glpi-project/glpi/blob/dbbfe91877e62e50beed3a5d21b3c3266e45fab2/src/Search.php#L4039-L4059

```suggestion
                            if ((isset($searchopt[$ID]["forcegroupby"]) && $searchopt[$ID]["forcegroupby"])) {
```